doc/release/1.11.0-notes.rst

```diff
@@ -118,3 +118,12 @@ c_contiguous arrays at some future time. A work around that is backward
 compatible is to use `a.T.view(...).T` instead. A parameter will also be
 added to the view method to explicitly ask for Fortran order views, but
 that will not be backward compatible.
+
+Invalid arguments to ```array.flatten```
```

Comment:
Oh, and don't worry about the test failure, seems there is something else going on.